index.js

```diff
@@ -1310,6 +1311,28 @@
   }
   S.fromMaybe = def('fromMaybe', {}, [a, $Maybe(a), a], fromMaybe);
 
+  //# fromMaybe_ :: (() -> a) -> Maybe a -> a
+  //.
+  //. Variant of [`fromMaybe`](#fromMaybe) which takes a thunk so the default
+  //. value is only computed if required.
+  //.
+  //. ```javascript
+  //. > global.fib = function fib(n) {
+  //. .   return n <= 1 ? n : fib(n - 2) + fib(n - 1);
+  //. . }
+  //. fib
```

Comment:
Perhaps if you wouldn't have to assert the return value of the function definition, it would read more easily, and it wouldn't look like a "stand-alone" example.